### DISCUSSION TITLE
What is the purpose of ignore_warnings?

### DISCUSSION THREAD

**Discussion by tgkolda** [sandialabs/pyttb#369] [General]
I'm trying to understand this segment of code in pyttb. Is this setting all warnings to be ignored? If so, why?

```
def ignore_warnings(ignore=True):
    """Helper to disable warnings"""
    if ignore:
        warnings.simplefilter("ignore")
    else:
        warnings.simplefilter("default")


ignore_warnings(True)
```

**Comment by ntjohnson1** [comment]
This is from very early days before the repo was open sourced so that history is lost to time. This specifically ignores the warning tool, but not logs printed at a warning level. If I had to guess we had a deprecation warning we silenced. We're probably safe to remove this or at least see what pops up when we remove it so we can label the rationale.

**Reply by ntjohnson1** [reply]
I removed it in #373 and didn't see any major issues (it at least helped avoid squashing some of our own warnings. We seem to use a mix of logging, print, and warnings which is a separate issue we should make consistent)
